Makefile.am

```diff
@@ -1,3 +1,6 @@
+ * (C) 2025 Telefónica Innovación Digital (laura.dominguez.cespedes@telefonica.com)
```

Comment:
Can we remove these comments from here -- this is not valid Makefile.am syntax and results in broken Makefiles when automake is run.